src/ThriftSQL.php

```diff
@@ -17,6 +17,17 @@ public function connect();
   */
   public function queryAndFetchAll( $queryStr );
 
+	/**
+	 * Get's a memory efficient iterator that you can use in a foreach loop.
+	 * If there's an error with the query, it will simply stop iterating.
+	 *
+	 * @param string $queryStr
+	 *
+	 * @return \ThriftSQL\ThriftStream
```

Comment:
Super minor but seeing as this is already namespaced within `\ThriftSQL` why name the iterator `\ThriftSQL\ThriftStream` and not something more obvious like simply `\ThriftSQL\Iterator` or `\ThriftSQL\ResultIterator`? After all it's a straight up implementation of `\Iterator`.

Reply:
Renamed to `\ThriftSQL\Utils\Iterator` in 519e794